python/ray/autoscaler/_private/command_runner.py

```diff
@@ -632,8 +632,9 @@ def run_rsync_up(self, source, target, options=None):
             self._get_docker_host_mount_location(
                 self.ssh_command_runner.cluster_name), target.lstrip("/"))
 
+        host_mount_location = os.path.dirname(host_destination.rstrip('/'))
         self.ssh_command_runner.run(
-            f"mkdir -p {os.path.dirname(host_destination.rstrip('/'))}",
+            f"mkdir -p {host_mount_location} && chown -R ubuntu {host_mount_location}",
```

Comment:
Should this be hard coded to ubuntu or the auth user in the cluster yaml?

Reply:
Good point, it should be the ssh user in the cluster yaml